opengrok-indexer/src/test/java/org/opengrok/indexer/history/GitRepositoryTest.java

```diff
@@ -61,12 +62,10 @@
 import static org.junit.jupiter.api.Assertions.assertNull;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.junit.jupiter.api.Assertions.assertTrue;
-import static org.opengrok.indexer.condition.RepositoryInstalled.Type.GIT;
 
 /**
  * @author austvik
  */
-@EnabledForRepository(GIT)
```

Comment:
Do we still have some other places which are requiring git command or can we remove the GIT check altogether?

Reply:
Yes, there are quite a few. I plan to revisit them separately. In fact, I added a item to https://github.com/oracle/opengrok/issues/1113#issuecomment-803470717